src/test/java/io/swagger/swaggerhub/plugin/SwaggerHubUploadTest.java

```diff
@@ -294,6 +290,64 @@ public void testMultiDefinitionsAreUploaded_andSCMSaveRequestMadeWithAccountPATA
+                SCM_ENABLE_INTEGRATION, SCM_REPOSITORY, SCM_REPOSITORY_OWNER, MULTI_UPLOAD_API_1_FILENAME, DefinitionFileFormat.YAML.getLanguageTarget(), FILE_FINDER_DIRECTORY, SCM_PERSONAL_ACCESS_TOKEN,
+                SCM_PROJECT, SCM_URL, SCM_PROJECT_COLLECTION);
+        RequestPatternBuilder saveSCMPluginRequestPattern2 = createPutSCMConfigRequestPatternWithUrlAndProjectCollection(API_OWNER, MULTI_UPLOAD_API_2_TITLE, MULTI_UPLOAD_API_2_VERSION, OAS2, SCM_BRANCH,
+                SCM_ENABLE_INTEGRATION, SCM_REPOSITORY, SCM_REPOSITORY_OWNER, MULTI_UPLOAD_API_2_FILENAME, DefinitionFileFormat.JSON.getLanguageTarget(), FILE_FINDER_DIRECTORY, SCM_PERSONAL_ACCESS_TOKEN,
+                SCM_PROJECT, SCM_URL, SCM_PROJECT_COLLECTION);
+        RequestPatternBuilder saveSCMPluginRequestPattern3 = createPutSCMConfigRequestPatternWithUrlAndProjectCollection(API_OWNER, MULTI_UPLOAD_API_3_TITLE, MULTI_UPLOAD_API_3_VERSION, OAS3, SCM_BRANCH,
+                SCM_ENABLE_INTEGRATION, SCM_REPOSITORY, SCM_REPOSITORY_OWNER, MULTI_UPLOAD_API_3_FILENAME, DefinitionFileFormat.YAML.getLanguageTarget(), FILE_FINDER_DIRECTORY, SCM_PERSONAL_ACCESS_TOKEN,
+                SCM_PROJECT, SCM_URL, SCM_PROJECT_COLLECTION);
+
+        //WHEN
+        getSwaggerUpload("src/test/resources/testProjects/upload-multi-definitions-save-scm-plugins-with-url-and-project-collection.xml").execute();
+
+        //THEN
+        verify(1, postRequestedFor(uploadDefinitionRequest1));
+        verify(1, postRequestedFor(uploadDefinitionRequest2));
+        verify(1, postRequestedFor(uploadDefinitionRequest3));
+
+        verify(1, putRequestedFor(saveSCMPluginConfigurationRequest1));
+        verify(saveSCMPluginRequestPattern1);
+
+        verify(1, putRequestedFor(saveSCMPluginConfigurationRequest2));
+        verify(saveSCMPluginRequestPattern2);
+
+        verify(1, putRequestedFor(saveSCMPluginConfigurationRequest3));
+        verify(saveSCMPluginRequestPattern3);
+    }
+
+    @Test
+    public void testInputFileIsUploaded_andSCMSaveRequestMadeWithUrlAndProjectCollection() throws Exception {
+        //GIVEN
+        UrlPathPattern saveDefinitionRequest = stubSaveDefinitionRequest(API_OWNER, INPUT_FILE_API, INPUT_FILE_API_VERSION, IS_PRIVATE, OAS2, JSON, SWAGGERHUB_API_TOKEN);
+        UrlPathPattern saveSCMPluginConfigurationRequest = stubSaveSCMPluginConfigurationRequest(API_OWNER, INPUT_FILE_API, INPUT_FILE_API_VERSION, OAS2, SWAGGERHUB_API_TOKEN);
+
+        RequestPatternBuilder saveSCMConfigRequest = createPutSCMConfigRequestPatternWithUrlAndProjectCollection(API_OWNER, INPUT_FILE_API, INPUT_FILE_API_VERSION, OAS2, SCM_BRANCH,
+                SCM_ENABLE_INTEGRATION, SCM_REPOSITORY, SCM_REPOSITORY_OWNER, INPUT_FILE_FILENAME,
+                DefinitionFileFormat.JSON.getLanguageTarget(), TEST_RESOURCES_DIRECTORY, SCM_PERSONAL_ACCESS_TOKEN, SCM_PROJECT, SCM_URL, SCM_PROJECT_COLLECTION);
```

Comment:
@michaelmelodysmartbear : Remove references to TFS in the constant property names. We may have other SCM integrations in future which use those same properties

done